src/app/modules/admin-shared/dynamic-form/main-form.component.html

```diff
@@ -1,15 +1,31 @@
 <div [formGroup]="form">
```

Comment:
why do we need main-form.html and dynamic form .htm inside dynamic form

Reply:
Removed